laba5.cpp

```diff
@@ -0,0 +1,114 @@
+#include <iostream>
+#include <string>
+#include <vector>
+ 
+enum lol {
+    Isotype,
+    Radioactive,
+    Ion,
+    Antimatter,
+    Stable,
+};
+ 
+class Atom
+{
```

Comment:
Your code must be wrapped in class by 1 tab or 4 spaces

Reply:
No